src/workers/parser/tests/__snapshots__/getSymbols.spec.js.snap

```diff
@@ -423,9 +517,9 @@ memberExpressions:
 [(19, 38), (19, 39)] [(19, 32), (19, 39)] a.b.c.v v
 [(19, 36), (19, 37)] [(19, 32), (19, 37)] a.b.c c
 [(19, 34), (19, 35)] [(19, 32), (19, 35)] a.b b
-[(27, 29), (27, 43)] [(27, 13), (27, 43)]  secondProperty
-[(27, 27), (27, 28)] [(27, 13), (27, 28)]  c
-[(27, 18), (27, 24)] [(27, 13), (27, 24)] obj2.doEvil doEvil
+[(23, 29), (23, 43)] [(23, 13), (23, 43)]  secondProperty
```

Comment:
why are some of these values different? is it because you removed some stuff from one of the fixtures?

Reply:
I moved some code to destructuring file, so the code is shifted by 4 lines, so the line number is now four les